packages/api/tests/lambdas/sf-scheduler.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const rewire = require('rewire');
+const sinon = require('sinon');
+const test = require('ava');
+
+const { SQS } = require('@cumulus/ingest/aws');
+const schedule = rewire('../../lambdas/sf-scheduler');
+
+const queueName = 'batman';
+const keyedItem = 'robin';
+const fakeS3Response = {
+  meta: {
+    queues: {
+      [queueName]: keyedItem,
+      startSF: 'startSF'
+    }
+  }
+};
+
+const Bucket = 'my-bucket';
+const Key = 'my-key';
+const restoreParseS3Uri = schedule.__set__('parseS3Uri', () => ({ Bucket, Key }));
+const restoreGetS3Object = schedule.__set__('getS3Object', () => Promise.resolve({
+  Body: JSON.stringify(fakeS3Response)
+}));
+
+const scheduleEventTemplate = {
+  meta: {},
+  cumulusMeta: {},
+  payload: {},
+  template: 's3://somewhere/nice'
+};
+
+const sqsStub = sinon.stub(SQS, 'sendMessage');
+
+test.afterEach(() => {
+  sqsStub.resetHistory();
+});
+
+test.after.always(() => {
+  restoreParseS3Uri();
```

Comment:
I'm not sure if you need to restore rewire stubs after the tests, do you? But I guess it doesn't hurt anything